### ISSUE TITLE
Error uploading

### ISSUE THREAD

**Issue by flofik228** [ArchiveTeam/seesaw-kit#146]
Tried to work on different projects and have same problem. Can't upload. It's retrying and retrying but I don't understand why can't it upload. Is that problem on my side or problem on server side?

**Comment by flofik228** [comment]
If needed any pictures or logs let me know.

**Comment by mnyc32** [comment]
@flofik228 did you ever figure this out? I'm having the same issue I think:
"Failed RsyncUpload for item"
"Retrying after 60 seconds..."

this just cycles. Thanks!

**Comment by TheTechRobo** [comment]
@mnyc32 This is currently known, there's some issues with the upload servers. It's being worked on and should hopefully be fixed soon.

**Comment by mnyc32** [comment]
@TheTechRobo Ah gotcha. Do you happen to know how long the issue has been known? 
I'm eager to contribute to the Archive Warrior project of backing up the US Govt pages before more data is taken down. Thanks!

**Comment by TheTechRobo** [comment]
It's been known for a few days now. Not sure exactly how long, though.

**Comment by flofik228** [comment]
Well I reported this 2 years ago.

**Comment by TheTechRobo** [comment]
Yes, it's known to be intermittent because IA can't always keep up with our uploads. But in the last few days, it has been practically constant because of technical issues with the upload targets.

Max connection errors are normal behaviour to see as long as it eventually uploads. The current situation is an outlier.

**Comment by flofik228** [comment]
So my bug was quite a normal thing to happen sometimes? Also if I keep it on trying will it eventually connect me and let me upload? (I haven't contributed to the project since I reported the problem and didn't get an answer so I might comeback).

**Comment by TheTechRobo** [comment]
Yes, it is normal (assuming the error message was indeed about max connections; if you see something else, please let us know!). The Warrior web UI only keeps finished tasks on the screen for a few seconds, so it might not be obvious that it's completing any work if you check it periodically. You can see if your Warrior is completing items by checking if the numbers next to your name on the leaderboard are increasing over time.

**Comment by mnyc32** [comment]
This is what I'm getting: https://imgur.com/a/hSGNcnd

Not sure if that's the same issue or not, TBH @TheTechRobo appreciate the help!

**Comment by Arkiver2** [comment]
Attaching previously linked image.

<img width="1667" height="665" alt="Image" src="https://github.com/user-attachments/assets/e70d8a76-f31c-46c8-bd4b-9a7e62956f24" />

**Comment by Arkiver2** [comment]
Closing as not relevant to seesaw-kit.
